lib/capistrano/tasks/raygun-deployment.cap

```diff
@@ -13,46 +13,50 @@ namespace :deploy do
 		ask(:raygun_auth_token, 'Required')
 	end
 
-	run_locally do
-
-        yaml = YAML::load_file(fetch(:raygun_release_path, 'RELEASE'))
+    set :raygun_api_key, [fetch(:raygun_api_key)] if !fetch(:raygun_api_key).kind_of?(Array)
 
-        use_git = fetch(:scm) == :git && fetch(:use_git, true)
-        git_hash = if use_git && system('git rev-parse --verify HEAD')
-            `git rev-parse --verify HEAD`
-        else
-            ""
-        end
-
-        deployment = {
-            'apiKey' => fetch(:raygun_api_key),
-            'version' => yaml['version'],
-            'ownerName' => yaml['ownerName'],
-            'emailAddress' => yaml['emailAddress'],
-            'comment' => yaml['notes'],
-            'scmIdentifier' => git_hash,
-            'createdAt' => yaml['createdAt']
-        }
-
-		uri = URI.parse(fetch(:raygun_api_uri, "https://app.raygun.io"))
-		deploymentEndpoint = "/deployments?authToken=#{fetch(:raygun_auth_token)}"
-
-        http = Net::HTTP.new(uri.host, uri.port)
-        http.use_ssl = fetch(:raygun_use_ssl, true)
-        http.verify_mode = OpenSSL::SSL::VERIFY_NONE
-        request = Net::HTTP::Post.new(deploymentEndpoint)
-        request.add_field('Content-Type', 'application/json')
-        request.body = deployment.to_json
-        res = http.request(request)
-        case res
-        when Net::HTTPSuccess
-          # OK
-            info "Sent deployment to Raygun"
-        else
-            raise "Error sending deployment to Raygun: " + res.value
+	run_locally do
+        fetch(:raygun_api_key).each do |raygun_api_key|
+
+            yaml = YAML::load_file(fetch(:raygun_release_path, 'RELEASE'))
+
+            use_git = fetch(:scm) == :git && fetch(:use_git, true)
+            git_hash = if use_git && system('git rev-parse --verify HEAD')
+                `git rev-parse --verify HEAD`
+            else
+                ""
+            end
+
+            deployment = {
+                'apiKey' => raygun_api_key,
+                'version' => yaml['version'],
+                'ownerName' => yaml['ownerName'],
+                'emailAddress' => yaml['emailAddress'],
+                'comment' => yaml['notes'],
+                'scmIdentifier' => git_hash,
+                'createdAt' => yaml['createdAt']
+            }
+
+    		uri = URI.parse(fetch(:raygun_api_uri, "https://app.raygun.io"))
+    		deploymentEndpoint = "/deployments?authToken=#{fetch(:raygun_auth_token)}"
+
+            http = Net::HTTP.new(uri.host, uri.port)
```

Comment:
Suggestion would be to include something like [httparty](http://jnunemaker.github.com/httparty) here in a separate `RaygunClient` class, this would outsource a bunch of the handling over to that.